sea.js

```diff
@@ -563,7 +582,7 @@
         bufiv = shim.Buffer.from(json.iv, opt.encode || 'base64');
         bufct = shim.Buffer.from(json.ct, opt.encode || 'base64');
         var ct = await aeskey(key, buf, opt).then((aes) => (/*shim.ossl ||*/ shim.subtle).decrypt({  // Keeping aesKey scope as private as possible...
-          name: opt.name || 'AES-GCM', iv: new Uint8Array(bufiv)
+          name: opt.name || 'AES-GCM', iv: new Uint8Array(bufiv), tagLength: 128
```

Comment:
Should this magic constant `128` became from SEA constants/module props?

Reply:
dono, It didnt work in react-native (ie msrcrypto lib) so i looked at their examples and it was used there 128.